client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -90,11 +91,17 @@ class TransferDomainPrecheck extends React.Component {
 	};
 
 	refreshStatus = () => {
-		this.props.refreshStatus( this.statusRefreshed );
+		this.props.refreshStatus( this.statusRefreshed ).then( result => {
+			const isUnlocked = get( result, 'inboundTransferStatus.unlocked' );
```

Comment:
What if this is an error? It's going to be `undefined`. Might want to treat that that differently.
<img width="199" alt="screen shot 2018-08-04 at 22 42 25" src="https://user-images.githubusercontent.com/1103398/43680295-af44a052-9837-11e8-8244-013c4d1028b1.png">

Reply:
Good catch. Thanks!